components/automate-gateway/api/iam/v2/request/users.proto

```diff
@@ -9,60 +9,53 @@ message CreateUserReq {
   option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
     json_schema: {
       required: ["id", "name", "password"]
-    }
+    },
+    example: { value: '{"name": "Display Name", "id": "username001rulez", "password": "aSafePassword"}'}
   };
+  // Unique ID. Cannot be changed. Used to log in.
```

Comment:
we've been trying to standardize to using _sign in_ instead of _log in_, would appreciate the same here and throughout if possible